### PR TITLE
Adding simple pagination to main page

### PR DESCRIPTION
**Pull request by tedlee**
Using Kaminari I've added simple "Next" and "Previous" button based pagination to the user's main page.

By default I've set posts per page at 10 but this can be edited within `app/controllers/posts_controller.rb` by changing the value of `.per(10)`

Simple pagination

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class PostsController < ApplicationController
   layout :choose_layout
 
   def index
-    @posts = Post.page(params[:page]).per(20)
+    @posts = Post.page(params[:page]).page(params[:page]).per(10)
```
Comment (jkap) [inline app/controllers/posts_controller.rb]: This is almost entirely unnecessary, as the page is already being pulled with a per-page count of 20.

**Comment by tedlee** [comment]
Woops, fixed the redundancy!

The current master doesn't properly handle pagination. When a user creates more than 20 posts, only the 20 newest posts are displayed. You can see this in effect here: http://electric-fog-7674.herokuapp.com/

This fixes that and adds buttons.

**Comment by jkap** [comment]
The current master handles pagination correctly in that manually appending ?page=2 to the end switches to the second page, but there is no easy way to do this.

The rest of the request looks great, I just figured you should know about the redundancy.

**Comment by tedlee** [comment]
Right on. Thanks for the heads up.

**Pull request merged**